src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -76,28 +76,38 @@ public static void setDatesToRepoConfigs(
      * Merges a {@code RepoConfiguration} from {@code repoConfigs} with another from {@code authorConfigs}
      * if {@code location} and {@code branch} matches.
```

Comment:
Just noticed, `location` and `branch` is not mentioned anywhere in this method.
Can update this javadoc to `if their {@code Repolocation} and branch matches`.